src/ui/prompts/environmentPrompts.ts

```diff
@@ -16,6 +16,12 @@ export type EnvironmentPromptResult = {
     environment: EnvironmentChoice['value']
 } & PromptResult
 
+const EnvironmentTypeValue: Record<string, number> = {
```

Comment:
how will this work if users have other envs/changed the keys?

Reply:
this is the type rather than the key, but that reminds me, we also have the disaster_recovery type

Reply:
oh true, good point